### PR TITLE
Invite permissions

### PR DESCRIPTION
**Pull request by weronica**
- Administrators can now specify permissions on the "Invite Administrator" page.
- When a new administrator goes to the site to register their account, they can also see the permissions that they have been granted. 
- Added helper functions to the `Permission` class.
- Wrote tests for `Permission` helper functions (`tests_permissions.py`).
- Removed unused files.

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -463,9 +466,51 @@ def add_resource_type(self, resource_type):
...
+		"Administer the site."       : 0x04,
+		"Invite new administrators." : 0x08 
+	}
+
+	@staticmethod
+	def get_dict():
+		"""Return a dictionary in which keys are Permission integers and values
+		are the public facing names of Permissions.
+		"""
+		return Permission.permission_phrases
```
Comment (jondubin) [inline app/models.py]: Will this return a dict where the keys are the public facing names and the values are the permission integers?

File: app/auth/views.py [hunk 2]
```diff
@@ -31,18 +31,21 @@ def invite_admin():
...
 				email=form.email.data,
 				organization_name=form.organization_name.data,
-				role_in_organization=form.role_in_organization.data
+				role_in_organization=form.role_in_organization.data,
+				permissions=Permission.get_int_from_list(form.permissions.data)
 			)
 			db.session.add(u)
 			db.session.commit()
 
+		print form.permissions.data
```
Comment (jondubin) [inline app/auth/views.py]: If you were using the `print` to test, it can be removed now

File: app/models.py (hunk 1 above)
Comment (weronica) [inline app/models.py]: Yes.

File: app/auth/views.py (hunk 2 above)
Comment (weronica) [inline app/auth/views.py]: Removed.

File: app/models.py (hunk 1 above)
Comment (jondubin) [inline app/models.py]: Is it switched in the docstring then?

Comment (weronica) [inline app/models.py]: Fixed.

**Pull request merged**